storage/replica.go

```diff
@@ -241,11 +247,11 @@ type Replica struct {
 	// raftMu protects Raft processing the replica.
 	//
 	// Locking notes: Replica.raftMu < Replica.mu
-	raftMu syncutil.Mutex
+	raftMu sync.Locker
```

Comment:
Not embedding a `syncutil.Mutex` means `go vet` won't warn about `Replica` being copied now. I'm not sure if that is a concern or not.